### DISCUSSION TITLE
How to get the strikethrough symbol of a font in excel?

### DISCUSSION THREAD

**Discussion by niuzhi** [qax-os/excelize#1451] [Q&A]
How to get the strikethrough symbol of a font in excel?
![image](https://user-images.githubusercontent.com/33443724/212632873-57f0dc9b-4973-4207-816a-bd6a1de43387.png)
![image](https://user-images.githubusercontent.com/33443724/212632946-e71c414a-c55e-49fc-aa6c-f660b8bfeb63.png)

**Comment by xuri** [comment]
Duplicate of #1450.
